### ISSUE TITLE
Navigation bar Issue #3

### ISSUE THREAD

**Issue by arundatin23** [LGazo-Babson/WebTechProject#3]
I think it would be better to add the contact form in your navigation bar instead of in the bottom right. It makes it more accessible for the viewer. Moreover, making an About Me page would be nice to read about the purpose of the website and why you choose this topic - it adds a personal touch.

**Comment by LGazo-Babson** [comment]
Yes, that is good insight and I will probably replace a page with an about me page. Thank you for all the time and consideration into helping improve my project.
